### ISSUE TITLE
Networking on main thread

### ISSUE THREAD

**Issue by patri9ck** [TheSilentPro/HeadDB#28]
**Describe the bug**
It does networking on the main thread which makes the whole server lagging.

**Information**
Plugin Version: 3.2.1

**Errors**
```
[20:14:48] [Paper Watchdog Thread/ERROR]:               java.base@16/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1653)
[20:14:48] [Paper Watchdog Thread/ERROR]:               java.base@16/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1577)
[20:14:48] [Paper Watchdog Thread/ERROR]:               java.base@16/sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:224)
[20:14:48] [Paper Watchdog Thread/ERROR]:               tsp.headdb.database.HeadDatabase.getHeadsNoCache(HeadDatabase.java:136)
[20:14:48] [Paper Watchdog Thread/ERROR]:               tsp.headdb.database.HeadDatabase.update(HeadDatabase.java:170)
[20:14:48] [Paper Watchdog Thread/ERROR]:               tsp.headdb.database.HeadDatabase.getHeads(HeadDatabase.java:117)
[20:14:48] [Paper Watchdog Thread/ERROR]:               tsp.headdb.api.HeadAPI.getHeads(HeadAPI.java:140)
[20:14:48] [Paper Watchdog Thread/ERROR]:               tsp.headdb.inventory.InventoryUtils.openDatabase(InventoryUtils.java:152)
[20:14:48] [Paper Watchdog Thread/ERROR]:               tsp.headdb.api.HeadAPI.openDatabase(HeadAPI.java:43)
[20:14:48] [Paper Watchdog Thread/ERROR]:               de.tim.jumpworld.command.SkullCommand.onCommand(SkullCommand.java:57)
```

**Comment by patri9ck** [comment]
It seems to be fixed in 4.0.1 but I'm not sure. If yes, this can be closed.

**Comment by TheSilentPro** [comment]
> It seems to be fixed in 4.0.1 but I'm not sure. If yes, this can be closed.

3.2.1 is an old version, download the latest (4.1.0) and see if its still an issue
